src/posix/platform/radio.cpp

```diff
@@ -94,18 +94,22 @@ void otPlatRadioSetPromiscuous(otInstance *aInstance, bool aEnable)
     SuccessOrDie(sRadioSpinel.SetPromiscuous(aEnable));
 }
 
-void platformRadioInit(const otPlatformConfig *aPlatformConfig)
+void platformRadioInit(otPosixRadioArguments *aArguments)
 {
+    ot::Posix::Arguments *args           = reinterpret_cast<ot::Posix::Arguments *>(aArguments);
+    bool                  resetRadio     = (args->GetValue("noreset") == NULL);
```

Comment:
@jwhui @abtink May I have your input about the parameter name style, should the name include `-` or not? ```suggestion
    bool                  resetRadio     = (args->GetValue("no-reset") == NULL);
```

Reply:
@jwhui @abtink Friendly reminder